commercial/app/views/jobs/jobsBlended.scala.html

```diff
@@ -0,0 +1,14 @@
+@(job: model.commercial.jobs.Job, clickMacro: Option[String])
+
+<div class="advert__blended-container advert__blended-container--jobs">
+    <div class="advert__blended-container__title">
```

Comment:
This is difficult. In theory you should think about creating a separate object `blended-component` so you can avoid having element of the element.

In the logic of BEM there are two ways:
`advert__body--blended`

or 

`blended__body`

That's why we have `gimbap-wrap` component for wrapping and then `gimbap` for inside and `gimbap-logo` for handling logos etc. Each is a separate object with it's own elements which you can create modifications of.

When you start creating elements of the elements it get's complicated. At least that's my experience.